introduction.md

```diff
@@ -0,0 +1 @@
+"Hello! I am Martin and I really want to learn something new."
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.